src/azure/cli/commands/__init__.py

```diff
@@ -89,7 +89,6 @@ def _delay(self):
         time.sleep(self.poll_interval_ms / 1000.0)
 
     def __call__(self, poller):
-        logger.warning(self.start_msg)
```

Comment:
'WARNING:' is not displayed in interactive mode (if there's a TTY available).
But log message are prefixed with the log level if there is no TTY (e.g. redirecting the output to file) as was the case in #572.